src/_backend_agg.cpp

```diff
@@ -212,8 +212,8 @@ agg::rect_i RendererAgg::get_content_extents()
 
     r.x1 = std::max(0, r.x1 - 1);
     r.y1 = std::max(0, r.y1 - 1);
-    r.x2 = std::max(r.x2 + 1, (int)width);
-    r.y2 = std::max(r.y2 + 1, (int)height);
+    r.x2 = std::min(r.x2 + 1, (int)width);
+    r.y2 = std::min(r.y2 + 1, (int)height);
```

Comment:
Nah, neither of them make sense to me. Instead, I think the -1 shouldn't be there for x1/y1, and instead of +1 on x2/y2, it should be -1 on width/height. Note, I am entirely certain if the subsequent code assumes 0 or 1-based indexing. I presume it is 0-based.